### PR TITLE
Fix typography docs

### PR DESCRIPTION
**Pull request by dgtlntv**
## Done

Fixed some of the typography docs to reflect the css more accuratly

## QA

- Look at the typography docs: https://vanilla-framework-5055.demos.haus/docs/base/typography
- Look at vanilla-framework/scss/_base_typography-definitions.scss, vanilla-framework/scss/_settings_font.scss and vanilla-framework/scss/_settings_spacing.scss
- Compare and make sure the typography docs are correct

### CODE REVIEW

**Comment by bartaz** [comment]
Thanks @dgtlntv, looks good from my point of view.

@lyubomir-popov could you have a look if proposed changes are good?

File: templates/docs/base/typography.md [hunk 1]
```diff
@@ -103,9 +107,14 @@ Our type scale consists of 5 font sizes, expressed as rems (root em units). For
       <p class="p-heading--1" style="font-size: 2.5rem; line-height: 3rem">H1 heading</p>
     </td>
   </tr>
+  <tr>
+    <td>64px</td>
+    <td>Ad hoc display headings for important bespoke pages on small and medium screens.</td>
```
Comment (lyubomir-popov) [inline templates/docs/base/typography.md:112]: why the inline styles?

**Comment by lyubomir-popov** [comment]
seems ok to me. Can I ask for a drive-by - to fix  the column widths of the th's here so these columns in the two tables align:
image

File: templates/docs/base/typography.md (hunk 1 above)
Comment (dgtlntv) [reply templates/docs/base/typography.md:112]: In order to make sure that the right size is displayed, independent of the screen you are currently on. But I am now realising, that we need to do it both ways actually. So also need to set inline styles for the large screen version.

Comment (bartaz) [reply templates/docs/base/typography.md:112]: I guess the question is what do we want to present here: all the available size options, regardless of screen size - even if we never use them on same screen size (for example, both sizes of h2 heading, both sizes of display heading) - I guess 8 options? - this requires examples with inline styles.

Or do we want to show how many options we have on given screen size (5?): so list 1 row for h2 heading, 1 row for display heading, but maybe mention that their size depends on screen size? (and keep examples responsive)

**Comment by dgtlntv** [comment]
> seems ok to me. Can I ask for a drive-by - to fix the column widths of the th's here so these columns in the two tables align: image

 I don't know how to properly do that in Vanilla. Is that something you could take care of @bartaz?

**Comment by bartaz** [comment]
> I don't know how to properly do that in Vanilla. Is that something you could take care of @bartaz?

Resizing columns is one of the most annoying things (especially when it requires to replace markdown table with HTML one…) ;) Consider it done :)

**Review by bartaz — APPROVED** [review]
looks good, thanks

**Pull request merged**